core/templates/dev/head/domain/objects/FractionObjectFactorySpec.js

```diff
@@ -38,6 +38,17 @@
        expect(new Fraction(false, 0, 0, 3).toString()).toBe('0');
      });
 
+     it('should return the correct integer part', function() {
+       expect(new Fraction(true, 1, 2 ,3).getIntegerPart()).toBe(-1);
+       expect(new Fraction(false, 1, 2 ,3).getIntegerPart()).toBe(1);
+       expect(new Fraction(true, 0, 2 ,3).getIntegerPart()).toBe(0);
+       expect(new Fraction(false, 0, 2 ,3).getIntegerPart()).toBe(0);
```

Comment:
nit: `, 2 ,3` -> `, 2, 3`

Reply:
Thanks, fixed!

(Btw, I'll need approval in order to merge.)